packages/backend/src/core/SequenceProcessor.ts

```diff
@@ -47,6 +49,7 @@ interface State {
 export class SequenceProcessor extends EventEmitter {
```

Comment:
contructor of this class is a bit unexpected for me, I thought that in our codebase we treat constructor as only property setting function. In this case inside the constructor `process()` is added to queque. IMO we should try to have the same behaviour repo-wise and change this flow to sth like: 
```
create object
start()
```

This should not be in the scope of this PR